src/Microsoft.AspNetCore.SignalR.Common/Internal/Protocol/MessagePackHubProtocol.cs

```diff
@@ -137,32 +148,41 @@ private void WriteMessageCore(HubMessage message, Stream output)
 
         private static void WriteInvocationMessage(InvocationMessage invocationMessage, Packer packer, Stream output)
         {
+            packer.PackArrayHeader(5);
```

Comment:
btw. @neuecc [predicted](https://github.com/aspnet/SignalR/pull/587#issuecomment-311338633) this...